QuizApp/Extensions/BackgroundGradient.swift

```diff
@@ -2,6 +2,20 @@ import UIKit
 
 class BackgroundGradient: CAGradientLayer {
 
+    override init() {
+        super.init()
+
+        setBackground()
+    }
+
+    override init(layer: Any) {
+        super.init(layer: layer)
+    }
```

Comment:
Is this needed?

Reply:
The application crashes without it when the simulator is put in landscape mode.